examples/notebooks/beam-ml/bigtable_enrichment_transform.ipynb

```diff
@@ -509,15 +509,48 @@
+        "Optionally, to provide additional configuration to the `BigTableEnrichmentHandler` handler, see [module docs](https://beam.apache.org/releases/pydoc/current/apache_beam.transforms.enrichment_handlers.bigtable.html#module-apache_beam.transforms.enrichment_handlers.bigtable)."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "yFMcaf8i7TbI"
+      },
+      "source": [
+        "**Note:** When exceptions occur, by default, the logging severity is set to warning ([`ExceptionLevel.WARN`](https://beam.apache.org/releases/pydoc/current/apache_beam.transforms.enrichment_handlers.utils.html#apache_beam.transforms.enrichment_handlers.utils.ExceptionLevel.WARN)).  To configure the severity to raise exceptions, set `exception_level` to [`ExceptionLevel.RAISE`](https://beam.apache.org/releases/pydoc/current/apache_beam.transforms.enrichment_handlers.utils.html#apache_beam.transforms.enrichment_handlers.utils.ExceptionLevel.RAISE). To ignore exceptions, set `exception_level` to [`ExceptionLevel.QUIET`](https://beam.apache.org/releases/pydoc/current/apache_beam.transforms.enrichment_handlers.utils.html#apache_beam.transforms.enrichment_handlers.utils.ExceptionLevel.QUIET).\n",
         "\n",
-        "The default `encoding` type is `utf-8`.\n",
+        "The following example demonstrates how to set exception level in `BigTableEnrichmentHandler`:\n",
         "\n",
-        "\n"
+        "```\n",
+        "bigtable_handler = BigTableEnrichmentHandler(project_id=PROJECT_ID,\n",
+        "                                             instance_id=INSTANCE_ID,\n",
+        "                                             table_id=TABLE_ID,\n",
+        "                                             row_key=row_key,\n",
+        "                                             exception_level=ExceptionLevel.RAISE)\n",
+        "```"
       ]
     },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "The key component of the Bigtable handler is the `row_key` parameter. The `row_key` represent the field in input schema (`beam.Row`) that contains the row key for a row in the table.\n",
+        "\n",
+        "As of Apache Beam version 2.54.0, if the table uses composite row keys, then you can:\n",
+        "* modify the input schema to contain the row key in the format required by Bigtable.\n",
+        "* use a custom enrichment handler ([example handler with composite row key support](https://gist.github.com/riteshghorse/21f4480c1c545ea01166e6bdf4c183e1))."
```

Comment:
```suggestion
        "* Use a custom enrichment handler. For more information, see the [example handler with composite row key support](https://gist.github.com/riteshghorse/21f4480c1c545ea01166e6bdf4c183e1)."
```

Reply:
Should this link be pointing to your repo?

Reply:
Actually I can replace this with a online pastebin.